### PR TITLE
Fixes #26237 - Add deploy by option

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by ares** [comment]
It seems we're lacking this info on API side, I opened https://projects.theforeman.org/issues/26373

While this is not blocking this PR, it would be good to get API support in first.

**Comment by xprazak2** [comment]
Opened https://github.com/theforeman/foreman_openscap/pull/370 to fix that.

File: lib/hammer_cli_foreman_openscap/policy.rb [hunk 1]
```diff
@@ -22,6 +22,7 @@ class InfoCommand < HammerCLIForeman::InfoCommand
         field :scap_content_profile_id, _("SCAP Content profile Id")
         field :tailoring_file_id, _("Tailoring file Id")
         field :tailoring_file_profile_id, _("Tailoring file profile Id")
+        field :deploy_by, _("Deployment options: 'ansible', 'puppet' or 'manual'")
```
Comment (ares) [inline lib/hammer_cli_foreman_openscap/policy.rb]: this is for info command only, no need to list options here, it should say singular "Deployment option"

here's the example with current version

```
Id:                                                  19
Name:                                                ansible
Created at:                                          2019-03-15 10:14:58 UTC
Period:                                              monthly
Weekday:                                             
Day of month:                                        1
Cron line:                                           
SCAP content Id:                                     51
SCAP Content profile Id:                             
Tailoring file Id:                                   
Tailoring file profile Id:                           
Deployment options: 'ansible', 'puppet' or 'manual': ansible
Locations:                                           
    Brno
Organizations:                                       
    Ares
Hostgroups:                                          
    ansible
```

**Comment by xprazak2** [comment]
Updated.

**Comment by ares** [comment]
Thanks @xprazak2, merging!

**Pull request merged**